### PR TITLE
Qualified import of `PropositionalEquality` etc. fixing #2280

### PR DESCRIPTION
**Pull request by jamesmckinna**

### CODE REVIEW

File: src/Relation/Binary/Indexed/Heterogeneous/Core.agda [hunk 1]
```diff
@@ -14,7 +14,7 @@ module Relation.Binary.Indexed.Heterogeneous.Core where
 open import Level
 import Relation.Binary.Core as B
 import Relation.Binary.Definitions as B
-import Relation.Binary.PropositionalEquality.Core as P
+import Relation.Binary.PropositionalEquality.Core as ≡
```
Comment (JacquesCarette) [inline src/Relation/Binary/Indexed/Heterogeneous/Core.agda:17]: Is this import even needed if you didn't need to switch anything?

File: src/Relation/Binary/Indexed/Heterogeneous/Bundles.agda [hunk 2]
```diff
@@ -14,7 +14,7 @@ module Relation.Binary.Indexed.Heterogeneous.Bundles where
 open import Function.Base
 open import Level using (suc; _⊔_)
 open import Relation.Binary.Core using (_⇒_)
-open import Relation.Binary.PropositionalEquality.Core as P using (_≡_)
```
Comment (JacquesCarette) [inline src/Relation/Binary/Indexed/Heterogeneous/Bundles.agda:17]: I don't think this import is needed at all.

File: src/Relation/Binary/Indexed/Heterogeneous/Definitions.agda [hunk 3]
```diff
@@ -14,7 +14,7 @@ module Relation.Binary.Indexed.Heterogeneous.Definitions where
 open import Level
 import Relation.Binary.Core as B
 import Relation.Binary.Definitions as B
-import Relation.Binary.PropositionalEquality.Core as P
+import Relation.Binary.PropositionalEquality.Core as ≡
```
Comment (JacquesCarette) [inline src/Relation/Binary/Indexed/Heterogeneous/Definitions.agda:17]: This import doesn't seem to be needed either.

File: src/Relation/Binary/Indexed/Homogeneous/Bundles.agda [hunk 4]
```diff
@@ -16,7 +16,7 @@ open import Function.Base using (_⟨_⟩_)
 open import Level using (Level; _⊔_; suc)
 open import Relation.Binary.Core using (_⇒_; Rel)
 open import Relation.Binary.Bundles as B
-open import Relation.Binary.PropositionalEquality.Core as P using (_≡_)
+open import Relation.Binary.PropositionalEquality.Core as ≡ using (_≡_)
```
Comment (JacquesCarette) [inline src/Relation/Binary/Indexed/Homogeneous/Bundles.agda:19]: Same here - not needed.

File: src/Tactic/Cong.agda [hunk 5]
```diff
@@ -36,7 +36,7 @@ open import Data.Unit.Base            using (⊤)
 open import Data.Word.Base   as Word  using (toℕ)
 open import Data.Product.Base         using (_×_; map₁; _,_)
 
-open import Relation.Binary.PropositionalEquality.Core as Eq using (_≡_; refl; cong)
+open import Relation.Binary.PropositionalEquality.Core as ≡ using (_≡_; refl; cong)
```
Comment (JacquesCarette) [inline src/Tactic/Cong.agda:39]: Is the `as` even needed?

**Review by JacquesCarette — CHANGES_REQUESTED** [review]
Went for something like 105 tedious changes before hitting something that seemed like it could be improved. So close!

Hopefully your emacs-fu is strong because the tediousness of this is amazing.

**Comment by jamesmckinna** [comment]
Yes: `label:mindless` absolute apologies to anyone reviewing this. 
But thanks for your actual intelligence applied to this, and yes, the edits would improve.

**Review by JacquesCarette — APPROVED** [review]
Very nice set of improvements. Still tedious but no longer mindless!

**Pull request merged**